mozilla/bedrock
# Remove hero component where possible

Removed hero from lean-data and reimagine-open

## One-line summary
Remove hero component where possible 

## Issue / Bugzilla link
#11493 

## Checklist

- [x] `/media/css/mozorg/lean-data.scss`
- [x] `/media/css/foundation/reimagine-open.scss`
- [x] `/media/css/foundation/annual-report-2019-2020.scss`
- [x]  `/media/css/firefox/installer-help.scss`
- [x]  `/media/css/firefox/testflight.scss`
- [x]  `/media/css/firefox/channel.scss`
- [x]  `/media/css/firefox/sync.scss`
- [x]  `/media/css/firefox/unfck.de.scss`
- [x] `/media/css/firefox/firstrun/nightly.scss`
- [x]  `/media/css/firefox/privacy/book.scss`
- [x]  `/media/css/firefox/features/adblocker.scss`
- [x]  `/media/css/firefox/set-as-default/landing.scss`
- [x]  `/media/css/firefox/set-as-default/thanks.scss`
- [x]  `/media/css/firefox/new/basic/download.scss`
- [x]  `/media/css/firefox/mobile/mobile.scss`
- [x]  `/media/css/mozorg/about.scss`
- [x]  `/media/css/firefox/unfck.scss`
- [x]  `/media/css/contentful/mozilla/c-hero.scss` (this one we can remove after updatig contentful to use Split)
- [x]  `/media/css/contentful/firefox/c-hero.scss` (same as above)
- [x]  `/media/css/firefox/pocket.scss`
- [x]  `/media/css/mozorg/home/home.scss`
- [x]  `/media/css/mozorg/diversity/2021/diversity-2021.scss`
- [x]  `/media/css/newsletter/newsletter.scss`

## Testing
To test this work:
- [x] No terminal errors regarding the Hero component

View the following pages and compare to production to make sure replacement elements and custom css match:
- [ ] [Re-imagine Data](http://localhost:8000/en-US/foundation/reimagine-open/) vs [prod](https://mozilla.org/en-US/foundation/reimagine-open/)
- [ ] [Installer-help](http://localhost:8000/en-US/firefox/installer-help/) vs [prod](https://www.mozilla.org/en-US/firefox/installer-help/)
- [ ] [testflight](http://localhost:8000/en-US/firefox/ios/testflight/) vs [prod](https://www.mozilla.org/en-US/firefox/ios/testflight/)
- [ ] [channel](http://localhost:8000/en-US/firefox/channel/desktop/) vs [prod](https://www.mozilla.org/en-US/firefox/channel/desktop/)
- [ ] [unfck.de](http://localhost:8000/de/firefox/unfck/) vs [prod](https://mozilla.org/de/firefox/unfck/)
- [ ] [nightly/firstrun](http://localhost:8000/en-US/firefox/nightly/firstrun) vs [prod](https://www.mozilla.org/en-US/firefox/nightly/firstrun/)
- [ ] [privacy/book](http://localhost:8000/en-US/firefox/privacy/book/) vs [prod](https://www.mozilla.org/en-US/firefox/privacy/book/)
- [ ] [adblocker](http://localhost:8000/en-US/firefox/features/adblocker/) vs [prod](https://www.mozilla.org/en-US/firefox/features/adblocker/)
- [ ] [set-as-default/landing](http://localhost:8000/en-US/firefox/set-as-default/) vs [prod](https://www.mozilla.org/en-US/firefox/set-as-default/)
- [ ] [set-as-default/thanks](http://localhost:8000/en-US/firefox/set-as-default/thanks/) vs [prod](https://www.mozilla.org/en-US/firefox/set-as-default/thanks/)
- [ ] [base-download](http://localhost:8000/en-US/firefox/mac/) vs [prod](https://www.mozilla.org/en-US/firefox/mac/) - the change is in the base template so all of the download pages should see the change reflected (mac, windows, linux)
- [ ] [mozorg/about](http://localhost:8000/en-US/about/) vs [prod](https://www.mozilla.org/en-US/about/)
- [ ] [firefox/pocket](http://localhost:8000/en-US/firefox/pocket/) vs [prod](https://www.mozilla.org/en-US/firefox/pocket/)
- [ ] [home - rest of the world that arent served the contentful homepage](http://localhost:8000/) vs [prod](mozilla.org) - not sure the best way to render this page (I had to change the template that would render for "home_view" and comment out the mr2 promo switch)
- [ ] welcome pages 1 -9 http://localhost:8000/en-US/firefox/welcome/1 etc
